todo-backend/app/Http/Controllers/AuthController.php

```diff
@@ -13,9 +12,13 @@ class AuthController extends Controller
      *
      * @return void
      */
+
+    private $userService;
+
     public function __construct()
```

Comment:
`public function __construct(UserService $userService)`
i ispod 
`$this->userService = $userService`